src/pages/[invoiceID].js

```diff
@@ -0,0 +1,182 @@
+import {
+  Stack,
+  Typography,
+  Divider,
+  Button,
+  Box,
+  IconButton,
+} from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { invoiceData } from '../../InvoiceDatabase';
+import { useRouter } from 'next/router';
+
+function InvoiceDetailsPage() {
+  const router = useRouter();
+  const { invoiceID } = router.query;
+
+  const invoice = invoiceData.filter(
+    (invoice) => Number(invoiceID) === invoice.id
+  )[0];
+
+  const date = new Date(invoice.due);
+  const currentDate = new Intl.DateTimeFormat(['ban', 'id']).format(date);
+  const currentMonth = () => {
+    const date = new Date(invoice.due);
+    return new Intl.DateTimeFormat(['ban', 'id']).format(
+      date.setMonth(date.getMonth())
+    );
+  };
+  const nextMonth = () => {
+    const date = new Date(invoice.due);
+    return new Intl.DateTimeFormat(['ban', 'id']).format(
+      date.setMonth(date.getMonth() + 1)
+    );
+  };
```

Comment:
The same observation here as before, about reusable functions that do one thing, and how you name them.